mesheryctl/internal/cli/root/system/testdata/model/list.model.output.golden

```diff
@@ -0,0 +1,108 @@
+CATEGORY                      	MODEL                                	VERSION            
```

Comment:
This will change overtime - potentially daily. Do not hardcode this.

Reply:
Still not answered...

Reply:
You might need explanation for doing this here as this will be dynamic and change from time to time